BuildTasks/ImportModule.Task.ps1

```diff
@@ -1,4 +1,4 @@
-function ImportModule
+function Import-BuildModule
```

Comment:
I wouldn't normally name internal functions with ```*-*``` syntax, but thats personal preference. Makes it confusing as it looks like you're using a cmdlet when you're using a local function instead. You can leave this, but just pointing out why it was named as it was originally.